src/backend/db/categories.js

```diff
@@ -1,4 +1,9 @@
 import { v4 as uuid } from 'uuid';
+import E1 from '../../assets/images/earings/E1.jpg';
+import N1 from '../../assets/images/necklace/N1.jpg';
+import R1 from '../../assets/images/rings/R1.jpg';
+import W1 from '../../assets/images/wedding/wed1.jpg';
+import J1 from '../../assets/images/J.jpg';
```

Comment:
You could export these from images folder using `index.js` file and import here all together using names imports.

Reply:
Shared - all shared components, utils, etc. adding here!
.util - yes for utility files,
action - exporting all actions types  from here, so that it can be imported anywhere easily.... generally it will avoid spelling mistakes in actions types name and easy access to available actions in app.